src/app/backend/replicasetdetail.go

```diff
@@ -158,7 +162,7 @@ func DeleteReplicaSetWithPods(client *client.Client, namespace string, name stri
 }
 
 // Returns detailed information about service from given service
-func getServiceDetail(service api.Service) ServiceDetail {
+func GetServiceDetail(service api.Service) ServiceDetail {
```

Comment:
Do you know that starting with Capital letter means that this is public function in golang? Should this function be public? I dont think so. Only the function that returns details should be. And you can test private methods nevertheless.